Add release workflow (#2420)
dedis/cothority: PUBLISH.md

```
This pins the library to the latest available development version.

## Releasing a new version for e-voting conode

Tag the release using

```bash
git tag -s vX.Y.Z -m "Cothority Release vX.Y.Z"
```

> and push it to master with `git push origin vX.Y.Z`. A tag push would trigger

Reviewer: ```suggestion
and push it to master with `git push origin vX.Y.Z`. A tag push triggers
```

> the `release` action workflow with new release binaries available at

Reviewer: ```suggestion
the `release` action workflow which releases new binaries in
```
